source/features/git-checkout-pr.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import ClippyIcon from 'octicon/clippy.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {getCurrentBranch} from '../github-helpers';
+
+function handleMenuOpening({delegateTarget: dropdown}: delegate.Event): void {
+	dropdown.classList.add('rgh-git-checkout'); // Mark this as processed
+	const [, user, repository] = select<HTMLAnchorElement>('.commit-ref.head-ref a')!.pathname.split('/', 3);
+
+	const tabContainer = select('tab-container', dropdown)!;
```

Comment:
Selector too generic

Reply:
Not sure what to rename it to. It is the 'tab container' it contains all the tabs :)

Reply:
It is only selecting `tab-container` within the dropdown menu.

Which one of the following classes should I add? `mt-1 position-absolute Box box-shadow-medium text-gray-dark right-0`

Reply:
There are so many parents and siblings that can be used, find one. There's also the very specific child with `[action="/users/checkout-preference"]`

Reply:
Got it. Do` .after` instead of appending it

Reply:
Going to need help here. When I try to do `.after` the whole tab stops working.
Maybe I need a nights sleep 😴

Reply:
@fregante I tried again. It needs to be appended to the tab-container.

Reply:
Select(the specific item).closest(tab container)